ManifestData.cs

```diff
@@ -23,251 +31,332 @@ public class ManifestData {
       243520, 226087, 224995, 224531, 223249, 222477, 221394, 220996,
       220675, 220625, 220480
     };
+    // Possibly replace with:
+    // public static List<int> allVersions { get { return data.Keys.ToList(); } }
 
-    public static readonly int GEHENNA = 358470;
-    public static readonly int PROTOTYPE = 322022;
-    public static readonly List<int> depots = new List<int> {257516, 257519, 257511, 257515};
-
-    private Dictionary<int, Dictionary<int, Datum>> data;
-
-    // Helper function so that data can be private. This also helps the runtime avoid accidentally copying data[version] as an intermediate.
-    public Datum this[int version, int depot] {
-      get {return data[version][depot];}
-    }
-
-    public long GetDownloadSize(int version, Package package) {
-      if (package == Package.Main) {
-        long totalDownloadSize = 0;
-        foreach (var depot in depots) totalDownloadSize += this[version, depot].size;
-        return totalDownloadSize;
-      } else if (package == Package.Gehenna) {
-        return this[version, GEHENNA].size;
-      } else if (package == Package.Prototype) {
-        return this[version, PROTOTYPE].size;
-      } else {
-        return 0;
+    public static long GetDownloadSize(int version, Package package = Package.All) {

```

Comment:
I'd like to avoid default parameters, unless there's a common use case. In this case, since it's not used at all, I'd like to avoid it (since it can lead to bugs / unexpected behavior).